templates/mail/style.html

```diff
@@ -0,0 +1,289 @@
+<style>
```

Comment:
Yes, premailer failed to fetch a CSS file so I use django template includes to solve that